lib/smtp.c

```diff
@@ -176,12 +177,15 @@ const struct Curl_handler Curl_handler_smtps = {
 /* SASL parameters for the smtp protocol */
 static const struct SASLproto saslsmtp = {
   "smtp",                     /* The service name */
-  334,                        /* Code received when continuation is expected */
-  235,                        /* Code to receive upon authentication success */
-  512 - 8,                    /* Maximum initial response length (no max) */
   smtp_perform_auth,          /* Send authentication command */
   smtp_continue_auth,         /* Send authentication continuation */
-  smtp_get_message            /* Get SASL response message */
+  smtp_cancel_auth,           /* Cancel authentication */
+  smtp_get_message,           /* Get SASL response message */
+  512 - 8,                    /* Maximum initial response length */
```

Comment:
Maybe explain where 512 and 8 come from?

Reply:
Done.